docs/endpoint.md

```diff
@@ -9,6 +9,10 @@ A message sent by endpoint needs to be encrypted locally, which avoids message b
 
 ## How to build endpoint
 
+### Install flatcc compiler
+The endpoint use flatbuffer as serializer. The `flatcc` tool should be install at `/usr/bin/` or `/bin/` directory.
```

Comment:
If `flatcc` can be found via $PATH and/or other environment variables, the above statement is not valid.